isb_lib/core.py

```diff
@@ -176,6 +170,24 @@ def coreRecordAsSolrDoc(coreMetadata: typing.Dict) -> typing.Dict:
     return doc
 
 
+def coreRecordAsSolrDoc(transformer: Transformer) -> typing.Dict:
+    """
+    Args:
+        transformer: A Transformer instance containing the document to transform
+
+    Returns: The coreMetadata in solr document format, suitable for posting to the solr JSON api
+    (https://solr.apache.org/guide/8_1/json-request-api.html)
+    """
+    coreMetadata = transformer.transform()
+
+    last_updated = transformer.last_updated_time()
+    if last_updated is not None:
+        date_time = parsed_date(last_updated)
+        if date_time is not None:
+            coreMetadata["sourceUpdatedTime"] = datetimeToSolrStr(date_time)
+    return _coreRecordAsSolrDoc(coreMetadata)
```

Comment:
New wrapper method so we can automatically insert the new column in the SOLR schema.  I didn't add it to the transformed metadata as it wasn't part of the original spec, and 2 of the 4 collections don't have it.